rustls/Cargo.toml

```diff
@@ -50,6 +83,8 @@ log = { workspace = true, optional = true }
 # TODO: specify `once_cell` version in single place - as RECENTLY UPDATED in upstream RUSTLS
 # (It may be best to specify the `once_cell` features here, as may be needed for updates moving forward.)
 once_cell = { version = "1.20.3", default-features = false, features = ["alloc", "race"] }
+# EXPLICIT `portable-atomic` dependency REQUIRED for `unsafe-assume-single-core` feature
```

Comment:
I think this comment should have been for better clarity:

```diff
-# EXPLICIT `portable-atomic` dependency REQUIRED for `unsafe-assume-single-core` feature
+# EXPLICIT `portable-atomic` dependency REQUIRED for `critical-section` & `unsafe-assume-single-core` feature
```

(will likely be updated in upcoming PR #49 or #45 anyway)